docs/content/doc/usage/email-setup.en-us.md

```diff
@@ -15,19 +15,33 @@ menu:
 
 # Email setup
 
-- To use Gitea's built-in Email support, update the `app.ini` config file [mailer] section:
+To use Gitea's built-in Email support, update the `app.ini` config file [mailer] section:
 
+## Sendmail version 
+Use the operating system’s sendmail command instead of SMTP. This is common on linux servers.  
```

Comment:
```suggestion
Use the operating system’s sendmail command instead of SMTP. This is common on Linux servers.
```

Small nit :wink:

Reply:
The two space at end of line where explicit to break line.